### ISSUE TITLE
User guide enhancement suggestion

### ISSUE THREAD

**Issue by Roger-Shepherd** [embench/embench-iot#84]
I was caught out by the naming convention for Python modules. For a speed benchmark the documentation says:

> --target-module <target module>: This mandatory argument specifies a python module in the pylib directory with definitions of routines to run the benchmark.

I naïvely assumed that "specifies" meant the name of the python file (i.e. xyz.py), it doesn't it means the name of the module (i.e. xyzzy). The documentation might make this explicit for those unfamiliar with Python conventions.

**Comment by jeremybennett** [comment]
Good point. Would you like to submit a patch to make this clear. Thanks, Jeremy

**Comment by Roger-Shepherd** [comment]
Done as pull request.
